helm-charts/templates/serviceAccount/apk-cluster-role.yaml

```diff
@@ -30,32 +30,14 @@ rules:
     resources: [ "gateways/status" ]
     verbs: [ "get","patch","update" ]
   - apiGroups: ["dp.wso2.com"]
-    resources: ["apis","servicemappings","runtimeapis","scopes"]
+    resources: ["apis"]
```

Comment:
does scopes work fine after removing it?

Reply:
it's present in the same file in a different location, this is a redundant scope